packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [35.1.0]
+
+### Added
+
+- Added optional `topBid` type in `NftMetadata` ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Added optional `floorAsk` in `TokenCollection` type ([#4522](https://github.com/MetaMask/core/pull/4522))
+- Added linea mainnet support to nft detection supported networks ([#4515](https://github.com/MetaMask/core/pull/4515))
+- Added a batch call to get collections API in detectNfts function ([#4443](https://github.com/MetaMask/core/pull/4443))
+- Added a call to get collections API inside getNftInformationFromApi ([#4443](https://github.com/MetaMask/core/pull/4443))
+
+### Changed
+
```

Comment:
```suggestion

### Changed

- Fetch NFT collections data from the NFT-API `getCollections` API when calling the `detectNfts` method of `NftDetectionController`, and the `updateNftMetadata` and `watchNft` methods of `NftController`. ([#4443](https://github.com/MetaMask/core/pull/4443))
```

- Recategorized to `Changed`
- Replaced reference to private class method with consumer-facing changes
- Is there any additional information we can provide about this API?

Reply:
If the source of the `getCollections` API is expected to be very clear to the consumers of this package, then we shouldn't need to add more information. Looks like this is the case here since it appears the NFT-API being referenced was already in use.

Reply:
Apologies @MajorLift i'm not sure i see what you meant here ☝️  could you elaborate? 🙏 

If by "NFT-API being referenced was already in use" you were referring to [this](https://github.com/MetaMask/core/blob/562e6a1aa44f1aa66b794d41590afc53e14dbddc/packages/assets-controllers/src/NftDetectionController.ts#L546) API, then yes it was already being used. 
With the new NFT UI designs we wanted to fetch more data that were available on the [getCollections](https://github.com/MetaMask/core/blob/562e6a1aa44f1aa66b794d41590afc53e14dbddc/packages/assets-controllers/src/NftDetectionController.ts#L665) API and then display that data if it exists in the UI.

So we added the call to [getCollections](https://github.com/MetaMask/core/blob/562e6a1aa44f1aa66b794d41590afc53e14dbddc/packages/assets-controllers/src/NftDetectionController.ts#L665) api after calling [getUserNFts](https://github.com/MetaMask/core/blob/562e6a1aa44f1aa66b794d41590afc53e14dbddc/packages/assets-controllers/src/NftDetectionController.ts#L629) api to fill in any missing fields;

Reply:
My concern is that the consumer might not know where this new `getCollections` API is coming from without digging through the code. It's unclear whether we're pulling from opensea or some internal endpoint we have already been using, or if we're pulling from a completely new external service. "NFT-API" doesn't seem to offer much clarification, as there are quite a few services with that name. 

Just saying it would be nice to have something like "OpenSea `getCollections` API" explicitly stated in the entry, unless just "`getCollections` API" is clear enough for our downstream consumers (who may or may not work for MetaMask).

Reply:
Superseded by https://github.com/MetaMask/core/pull/4525#discussion_r1678246446